tcl/org.testeditor.tcl.dsl.ui/src/org/testeditor/tcl/dsl/ui/editor/DropTargetXtextEditorListener.xtend

```diff
@@ -34,44 +35,47 @@ class DropTargetXtextEditorListener extends DropTargetAdapter {
 
 	override void drop(DropTargetEvent event) {
 
-		val List<String> toFormat = newArrayList
-		val List<String> currentElement = newArrayList
+		val List<String> elementPathsToFormat = newArrayList
+		val insertedTestStepPath = new AtomicReference<String>
 
-		editor.document.modify[updateModel(toFormat, currentElement)]
-		editor.document.modify [
-			formatRelevantRegion(toFormat)
+		editor.document => [
+			modify[updateModel(elementPathsToFormat, insertedTestStepPath)]
```

Comment:
No, this would result in an exception: 
java.lang.IllegalStateException: Cannot modify document textually and semantically within the same unit of work.
We could however combine the last two actions into one modify, but that does not make it that much more readable, what do you think?